src/commcare_cloud/manage_commcare_cloud/list_vault_keys.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+from itertools import chain
+
+from commcare_cloud.commands.command_base import CommandBase
+from commcare_cloud.environment.main import get_environment
+from commcare_cloud.environment.paths import get_available_envs
+
+
+class ListVaultKeys(CommandBase):
+    command = 'list-vault-keys'
+    help = """
+    Audit the structure of vault.yml files.
+
+    A password will be required for each encrypted vault file in the env.
+    """
+
+    def run(self, args, unknown_args):
+        envs = sorted(get_available_envs(exclude_symlinks=True))
+        var_keys = {}
+        for env in envs:
+            print('[{}] '.format(env), end='')
```

Comment:
this print if for the password prompt (the prompt just says "`Vault Password:`" by default; I wanted it to include the name of the env that each prompt is for).